src/patternfly/components/Tabs/tabs-item.hbs

```diff
@@ -9,7 +9,10 @@
     {{/if}}
   {{/if}}
   {{~#if tabs-item--IsAdd}} pf-m-add{{/if}}
+  {{~#if tabs-item--IsOverflow}} pf-m-overflow{{/if}}
+  {{~#if tabs-item--IsOverflowExpanded}} pf-m-expanded{{/if}}
   {{~#if tabs-item--modifier}} {{tabs-item--modifier}}{{/if}}"
+  {{~#if tabs-item--IsOverflowExpanded}} aria-expanded="true"{{/if}}
```

Comment:
typically `aria-expanded` goes on the thing that toggles the expansion, so I'd move this to the button.